core/src/commonMain/kotlin/com/ustadmobile/core/controller/PersonEditPresenter.kt

```diff
@@ -2,38 +2,53 @@ package com.ustadmobile.core.controller
 
 import com.ustadmobile.core.db.UmAppDatabase
 import com.ustadmobile.core.generated.locale.MessageID
+import com.ustadmobile.core.impl.AppConfig
```

Comment:
I think for this to follow the crud pattern this EditPresenter would also need to use PersonWithAccount (to include the passwords etc) so that the view edits the object directly (instead of the presenter having to ask for other properties).